### DISCUSSION TITLE
Article seach function

### DISCUSSION THREAD

**Discussion by jonnyhoff** [dailydotdev/daily#415] [Feature Requests]
Firstly... thanks for the great app.

I'd love to be able to search articles on the "Popular" & "Most upvoted" pages.

**Comment by nimrodkra** [comment]
Hi @jonnyhoff 👋

Thanks for taking the time to write this feature request!
Excellent idea, we'll discuss it during our upcoming product planning. I'll keep you posted once there's a decision on it.
